app/Http/Controllers/Twill/BaseApiController.php

```diff
@@ -127,4 +363,17 @@ protected function indexItemData($item)
 
         return ['edit' => $editRoute];
     }
+
+    /**
+     * Option to setup links and the possibility of augmenting a model
+     */
+    protected function enableAugmentedModel(): void
+    {
+        $this->hasAugmentedModel = true;
+    }
+
+    protected function setIdColumnLabelPrefix(string $idColumnLabelPrefix): void
+    {
+        $this->idColumnLabelPrefix = $idColumnLabelPrefix;
+    }
```

Comment:
These were created to follow Twill's pattern of using setters for controller properties to be used in the `setUpController()` method.